robotiq_s_model_articulated_gazebo/launch/robotiq_gripper_empty_world.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="limited" default="false"/>
+
+  <!-- startup simulated world --> 
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <arg name="world_name" default="worlds/empty.world"/>
+    <arg name="paused" value="false"/>
+  </include>
+
+  <!-- send robot to urdf to param serv -->
+  <include file="$(find robotiq_s_model_visualization)/launch/robotiq_gripper_upload.launch">
+  </include>
+
+  <!-- Start the publisher for the left hand (the only hand) -->
+  <node pkg="robot_state_publisher" type="state_publisher" name="robotiq_hands_l_hand_robot_state_publisher">
+    <param name="publish_frequency" type="double" value="50.0" />
+    <remap from="joint_states" to="/robotiq_hands/left_hand/joint_states" />
```

Comment:
> The way it is structured is it actually checks if l_ or r_ is the prefix. That way, those who want to use 2 robotiq grippers can do so. I don't forsee anyone using more than 2 grippers on their robot in the future.

Exactly: it checks for the `l_` or `r_` prefix. My suggestion was to make it work with *any* prefix.

In general I like to avoid making any assumptions about the use of my components by others. And (artificially) limiting the number of instances of this macro to two seems like a rather serious assumption.

Removing it might be out-of-scope of this PR though.